packages/tables/src/Columns/Concerns/HasAlignment.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Filament\Tables\Columns\Concerns;
+
+trait HasAlignment
+{
+    protected ?string $alignment = null;
+
+    public function alignment(?string $alignment): static
```

Comment:
Can we also add some convenience methods like `alignLeft`, etc?

Reply:
done